bazel/osx/BUILD

```diff
@@ -0,0 +1,46 @@
+licenses(["notice"])  # Apache 2
+package(default_visibility = ["//visibility:public"])
+
+filegroup(
+    name = "srcs",
+    srcs = glob(["**"] + ["**/*"]),
+)
+
+exports_files([
+    "xcode_locator.m",
+])
+
+DARWIN_XCODE_LOCATOR_COMPILE_COMMAND = """
+  /usr/bin/xcrun clang -fobjc-arc -framework CoreServices \
+      -framework Foundation -o $@ $<
+"""
+
+genrule(
```

Comment:
FWIW, I could revert the change that enables the darwin compiler toolchain and turn that on in the PR with all the other necessary changes to get it compiling under OSX.